### ISSUE TITLE
Tides of Chaos usage

### ISSUE THREAD

**Issue by evilcj925** [trioderegion/dnd5e-helpers#2]
Can something be added that makes it roll on the surge table when Tides of Chaos is used? 
Maybe a check for the resource named Tides of Chaos, where if it is found to be used, have 0 charges for it, then it skips the weave d20 roll and rolls directly on the surge table? Or maybe add a button to the chat card of the weave roll that causes a surge table roll?

**Comment by trioderegion** [comment]
To be clear, is this intended for RAW 5e?

After reading Tides of Chaos again, it looks like it should recharge if a "forced" surge occurs. If I was to rephrase, the ideal solution would be:

A spell is cast. If the caster has 0 uses of Tides of Chaos remaining, then "The Weave" message should include a "Force Surge" button if this spell didnt surge. This force surge biutton rolls on the wild magic table AND gives 1 use of Tides of Chaos.

Just a development note, this would include a clickable button in a chat output -- something I have not done yet, but need to figure out.  I like this feature request though and will work on it.

**Comment by evilcj925** [comment]
Yes, Tides of Chaos recharges after a forced surge, or at a long rest.
